README.md

```diff
@@ -91,6 +93,10 @@
 
 ## Programming Languages
 
+### Crystal
+
+* [crystal2nix](https://github.com/nix-community/crystal2nix) - Convert shard.lock into Nix expressions.
```

Comment:
Add backticks/grave symbols around "shard.lock" for formatting consistency with other filenames like `poetry.lock`.